charts/nginx/templates/nginx-rolebinding.yaml

```diff
@@ -18,6 +18,6 @@ roleRef:
   name: {{ template "nginx.fullname" . }}
 subjects:
   - kind: ServiceAccount
-    name: {{ template "nginx.serviceAccount" . }}
+    name: {{ template "nginx.ServiceAccount" . }}
```

Comment:
This inconsistency between serviceAccount and ServiceAccount is bad. It looks like we have a lot of it in our chart already, but we should try as hard as possible to stick to convention, which is `camelCase`, not `PascalCase`.